nautobot/extras/views.py

```diff
@@ -543,12 +543,7 @@ def post(self, request, slug):
 
         repository = get_object_or_404(GitRepository.objects.all(), slug=slug)
 
-        # Allow execution only if RQ worker process is running
```

Comment:
Is there a comparable check we can make to ensure that the Celery worker is running, rather than just removing this check? Same question below.